ui/components/component-library/box/box.tsx

```diff
@@ -50,88 +49,84 @@ function isValidString(type: StyleDeclarationType, value: StylePropValueType) {
-    if (!value && typeof value !== 'number') {
-      return {};
-    }
-    const classNamesObject: ClassNamesObject = {};
-    // if value is an array with single item e.g. marginTop={[1]}
-    const singleArrayItemProp =
-      Array.isArray(value) && value.length === 1 ? value[0] : undefined;
-    // if value single value e.g. marginTop={1}
-    const singleValueProp =
-      (!Array.isArray(value) && typeof value === 'string') ||
-      typeof value === 'number'
-        ? value
-        : undefined;
-    // single digit equals single value or single array item
-    let singleValue;
-    if (singleValueProp || singleValueProp === 0) {
-      singleValue = singleValueProp;
-    }
-    if (singleArrayItemProp || singleArrayItemProp === 0) {
-      singleValue = singleArrayItemProp;
-    }
-    // 0 is an acceptable value but is falsy in js
-    if (singleValue || singleValue === 0) {
-      // add base style without any breakpoint prefixes to classObject
-      classNamesObject[
-        `${BASE_CLASS_NAME}--${styleDeclaration}-${singleValue}`
-      ] = validatorFn(styleDeclaration, singleValue);
-    } else if (Array.isArray(value)) {
-      // If array with more than one item
-      switch (value.length) {
-        case 4:
-          // add base/sm/md/lg
-          classNamesObject[
-            `${BASE_CLASS_NAME}--${styleDeclaration}-${value[0]}`
-          ] = validatorFn(styleDeclaration, value[0]);
-          classNamesObject[
-            `${BASE_CLASS_NAME}--${BREAKPOINTS[1]}:${styleDeclaration}-${value[1]}`
-          ] = validatorFn(styleDeclaration, value[1]);
-          classNamesObject[
-            `${BASE_CLASS_NAME}--${BREAKPOINTS[2]}:${styleDeclaration}-${value[2]}`
-          ] = validatorFn(styleDeclaration, value[2]);
-          classNamesObject[
-            `${BASE_CLASS_NAME}--${BREAKPOINTS[3]}:${styleDeclaration}-${value[3]}`
-          ] = validatorFn(styleDeclaration, value[3]);
-          break;
-        case 3:
-          // add base/sm/md
-          classNamesObject[
-            `${BASE_CLASS_NAME}--${styleDeclaration}-${value[0]}`
-          ] = validatorFn(styleDeclaration, value[0]);
-          classNamesObject[
-            `${BASE_CLASS_NAME}--${BREAKPOINTS[1]}:${styleDeclaration}-${value[1]}`
-          ] = validatorFn(styleDeclaration, value[1]);
-          classNamesObject[
-            `${BASE_CLASS_NAME}--${BREAKPOINTS[2]}:${styleDeclaration}-${value[2]}`
-          ] = validatorFn(styleDeclaration, value[2]);
-          break;
-        case 2:
-          // add base/sm
-          classNamesObject[
-            `${BASE_CLASS_NAME}--${styleDeclaration}-${value[0]}`
-          ] = validatorFn(styleDeclaration, value[0]);
-          classNamesObject[
-            `${BASE_CLASS_NAME}--${BREAKPOINTS[1]}:${styleDeclaration}-${value[1]}`
-          ] = validatorFn(styleDeclaration, value[1]);
-          break;
-        default:
-          console.log(`Invalid array prop length: ${value.length}`);
-      }
```

Comment:
Note that the body of this function has not change at all. The only thing that change was the removal of the `memoize` call that wrapped this function. That removal  changes indentation and so we have the many-lined diff.